src/inversion_scripts/calc_sensi.py

```diff
@@ -87,6 +87,21 @@ def calc_sensi(
         nlon = len(base_data["lon"])  # 52
         nlat = len(base_data["lat"])  # 61
         nlev = len(base_data["lev"])  # 47
+        base_var = base_data["SpeciesConcVV_CH4"] # Read base data before loop
+
+        # Save this data into numpy array so we don't need to read files in loop
+        pert_datas = []
+         # For each state vector element
+        for e in elements:
```

Comment:
is the reason for moving this out of process to avoid memory issues or something else?

Reply:
Meghan He added these lines in the commit titled "bug fixes for 2x2.5, parallelize jacobian and faster calc_sensi" so it would seem these lines were added to speedup that code.